github_rules/github_org_auth_modified.yml

```diff
@@ -0,0 +1,40 @@
+AnalysisType: rule
+Filename: github_org_auth_modified.py
+RuleID: GitHub.Org.AuthChange
+DisplayName: GitHub Org Authentication Method Changed
+Enabled: true
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: Critical
+SummaryAttributes:
+  - name
```

Comment:
I don't see a `name` field, should this be `actor`?

Reply:
This was brain fade. Fixed now.